onnxruntime/test/providers/cpu/tensor/unsqueeze_op_test.cc

```diff
@@ -63,5 +63,17 @@ TEST(TensorOpTest, Unsqueeze_OutOfRange) {
   test.Run(OpTester::ExpectResult::kExpectFailure, "Mismatch between number of source and target dimensions.");
 }
 
+// Disable this test given ONNX spec shape inference has bug.
+/*
```

Comment:
nit: prefer DISABLED_ prefix
[https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests)

Reply:
Thanks, re-enabled this test with latest onnx spec fix.